assets/address.html

```diff
@@ -19,7 +19,7 @@
     </select>
   </form>
 
-  <iframe src="http://fillr-code-test-x.s3.amazonaws.com/code-test-b/payment.html"></iframe>
+  <iframe src="http://localhost:9999/base/assets/payment.html"></iframe>
```

Comment:
Does this make the exercise easier, since the frame content is no longer cross-domain relative to other assets?

If so, maybe use `127.0.0.1` here.